Exclude jquery/fileUploader modules from bundle

jcarmona120 opened the pull request:
ConnectWise Ticket: #431730

## Issue Being Resolved:

```
bundle-common.min.js:1 Uncaught TypeError: (intermediate value)(intermediate value)(...) is not a function
    at bundle-common.min.js:1
```

The culprit for this error was the bundled jquery/fileUploader modules. For some reason in magepack's bundling process, these modules were bundled incorrectly with a missing semi-colon. For more details on when this error normally occurs, see [this StackOverflow article.](https://stackoverflow.com/questions/42036349/uncaught-typeerror-intermediate-value-is-not-a-function)

This error caused breaks on the PLP (loader would run forever when filter was applied) and Checkout (loader would run forever as page loaded). 

## Solution

Magepack does not have a configuration file that easily allows us to modify modules to be excluded so we had to fork our own version of magepack and hardcode our excluded modules to `lib/generate/excludedModules.js`.

In the future, we should find a way to exclude modules via a config file.

jcarmona120 committed:
Exclude jquery/fileUploader modules from bundle

Leland commented on lib/generate/excludedModules.js:
```diff
@@ -17,4 +17,18 @@ module.exports = [
     'exports',
     'require',
     'module',
+    'jquery/fileUploader/jquery.fileupload',
+    'jquery/fileUploader/jquery.fileupload-process',
+    'jquery/fileUploader/jquery.fileupload-video',
+    'jquery/fileUploader/jquery.fileupload-audio',
+    'jquery/fileUploader/jquery.fileupload-image',
+    'jquery/file-uploader',
+    'jquery/fileUploader/jquery.iframe-transport',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-meta',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-scale',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-orientation',
+    'jquery/fileUploader/vendor/blueimp-load-image/js/load-image-exif',
+    'jquery/fileUploader/vendor/blueimp-canvas-to-blob/js/canvas-to-blob',
+    'jquery/fileUploader/vendor/jquery.ui.widget',
```
We can just blanket exclude everything relating to the two, IMO. 

Please add a comment explaining what this is doing as well – be technical and specific (i.e. discuss the semicolon)

```suggestion
    'jquery/fileUploader/',
    'jquery/file-uploader',
```

Leland reviewed (changes requested):
Great, thanks!

Can you also update the PR description to note the specific console error?

jcarmona120 committed:
Simplify excluded bundles

These phrases are used in an includes(excludedModules.moduleName) so we can simplify which terms are used in array.

Co-authored-by: Leland Clemmons <leland.clemmons@gmail.com>
